lib/mr/factory/record_factory.rb

```
@@ -45,7 +45,12 @@ def apply_args(record, args = nil)
     def column_args
       @columns ||= non_association_columns(@record_class)
       @columns.inject({}) do |a, column|
-        a.merge(column.name.to_s => MR::Factory.send(column.type))
+        begin
+          column_type = column.type || column.sql_type.to_sym
+          a.merge(column.name.to_s => MR::Factory.send(column_type))
+        rescue StandardError
```

@Criptak - I think this is a little aggressive and it might hide errors. I'd prefer to either change this to `NoMethodError` or switch to an if.

For the `NoMethodError` I'd want to isolate generating the column value:

```ruby
column_type  = column.type || column.sql_type
column_value = begin
  MR::Factory.send(column_type)
rescue NoMethodError
  nil
end
a.merge(column.name.to_s => column_value)
```

This still has the problem of hiding no method errors in the factory methods themselves. For MR we test those individually so its not the biggest deal.

Alternatively, using an if:

```ruby
column_type  = column.type || column.sql_type
if MR::Factory.respond_to?(column_type)
  a.merge(column.name.to_s => MR::Factory.send(column_type))
else
  a
end
```

This doesn't hide any errors but I'm guessing it (generally) performs worse because of the call to `respond_to?` and the `if` (it may not perform worse than when the other implementation raises an error but that isn't the common case).

/cc @kellyredding